Purchases/Public/RCPurchasesErrorUtils.m

```diff
@@ -146,6 +150,8 @@ static RCPurchasesErrorCode RCPurchasesErrorCodeFromRCBackendErrorCode(RCBackend
             return RCInvalidAppleSubscriptionKeyError;
         case RCBackendUserIneligibleForPromoOffer:
             return RCIneligibleError;
+        case RCBackendInvalidSubscriberAttributes:
+            return RCBackendInvalidSubscriberAttributesError;
```

Comment:
We normally try to map these backend errors to an error part of the `RCPurchasesErrorCode` enum, like in:
```
case RCBackendStoreProblem:
        case RCBackendPlayStoreQuotaExceeded:
        case RCBackendPlayStoreInvalidPackageName:
        case RCBackendPlayStoreGenericError:
            return RCStoreProblemError;
```
The reason behind that is that we expose these errors and we don't want the developers to have to handle all these backend errors separately. 

I think in this case we have to create one, which you already did 😄. We don't have one `RCPurchasesErrorCode` already that could group the backend errors when posting user attributes. I would change `RCBackendInvalidSubscriberAttributesError` to `RCInvalidSubscriberAttributesError` though to differentiate that error from the backend one, so that we can asign future `RCBackendErrorCode` related to attributes to that `RCPurchasesErrorCode.RCInvalidSubscriberAttributesError`.

Reply:
sounds good, will do. thanks!